bin/ytdl.js

```diff
@@ -366,10 +369,16 @@ if (opts.infoJson) {
       var dataRead = 0;
       readStream.on('data', (data) => {
         dataRead += data.length;
-        if (dataRead === size) {
-          updateBar();
-        } else {
-          updateBarThrottled();
+
+        if (is_stdout_mutable) {
+          if (dataRead === size) {
+            updateBar();
+          } else {
+            updateBarThrottled();
+          }
+        }
+        else if (dataRead === size) {
+          console.log(`download complete: ${size} bytes of data read`)
```

Comment:
I don't think we need the size of the download here, since it's already printed above.

But it might be useful if the stresm is a live stream, and it's ended at some point